WTWSMS/interface/portrait_properties.txt

```diff
@@ -0,0 +1,1389 @@
+			NOT = { martial = 12 }
+			NOT = {
+				primary_title = {
+					OR = {
+						title = d_knights_hospitaler
+						title = d_knights_templar
+						title = d_teutonic_order
+					}
+				}
+			}
+		}
+		modifier = {
+			factor = 0
+			OR = {
+				is_female = yes
+				real_tier = king
+				real_tier = emperor
+				AND = {
+					real_tier = duke
+					any_demesne_title = {
+						real_tier = baron
+					}
+					NOT = {
+						primary_title = {
+							OR = {
+								title = d_knights_hospitaler
+								title = d_knights_templar
+								title = d_teutonic_order
+							}
+						}
+					}
+				}
+				AND = {
+					religion_group = christian
+					OR = {
+						AND = {
+							is_ruler = yes
+							is_theocracy = yes
+						}
+						has_job_title = job_spiritual
+					}
+				}
+			}
+		}
+	}
+	
+	# Ragged prison clothes
+	11 = {
+		factor = 100
+		modifier = {
+			factor = 0
+			prisoner = no
+		}
+	}
+}
+
+# p4 beard
+4 = {
+	0 = {
+		factor = 1
+		modifier = {
+			factor = 0
+			religion_group = muslim
+			NOT = { graphical_culture = muslimgfx }
+		}
+		modifier = {
+			factor = 30
+			religion_group = christian
+			OR = {
+				culture = greek
+				culture = italian
+			}
+			OR = {
+				graphical_culture = southerngfx
+				graphical_culture = byzantinegfx
+			}
+		}
+		modifier = {
+			factor = 15
+			religion_group = christian
+			culture_group = iberian
+			OR = {
+				graphical_culture = southerngfx
+				graphical_culture = byzantinegfx
+			}
+		}
+	}
+	1 = {
+		factor = 1
+	}
+	2 = {
+		factor = 1
+	}
+	3 = {
+		factor = 1
+	}
+	4 = {
+		factor = 1
+	}
+	5 = {
+		factor = 1
+	}
+	6 = {
+		factor = 1
+	}
+	7 = {
+		factor = 1
+	}
+	8 = {
+		factor = 1
+	}
+	9 = {
+		factor = 1
+		modifier = {
+			factor = 5
+			religion_group = christian
+			culture_group = iberian
+			OR = {
+				graphical_culture = southerngfx
+				graphical_culture = byzantinegfx
+			}
+		}
+		modifier = {
+			factor = 5
+			OR = {
+				culture = roman
+				culture = romano_gallic
+				culture = romano_hispanic
+				culture = thraco_roman
+				culture = illyro_roman
+				culture = latin
+			}
+			graphical_culture = romangfx
+		}
+	}
+	10 = {
+		factor = 1
+		modifier = {
+			factor = 0
+			religion_group = muslim
+			NOT = { graphical_culture = muslimgfx }
+		}
+		modifier = {
+			factor = 20
+			religion_group = christian
+			culture_group = iberian
+			OR = {
+				graphical_culture = southerngfx
+				graphical_culture = byzantinegfx
+			}
+		}
+		modifier = {
+			factor = 60
+			religion_group = christian
+			OR = {
+				culture = greek
+				culture = italian
+			}
+			OR = {
+				graphical_culture = southerngfx
+				graphical_culture = byzantinegfx
+			}
+		}
+		# romangfx index 10 is "no beard"
+		modifier = {
+			factor = 80
+			OR = {
+				culture = roman
+				culture = romano_gallic
+				culture = romano_hispanic
+				culture = thraco_roman
+				culture = illyro_roman
+				culture = latin
+			}
+			graphical_culture = romangfx 
```

Comment:
Here's the addition from vanilla portrait_properties.
It makes much more likely that Romans will not grow beards, as the Mediterranean long beards don't feel very Roman to me (but maybe that's just me :p)